include/swift/Option/FrontendOptions.td

```diff
@@ -381,6 +381,10 @@ def enable_experimental_static_assert :
   Flag<["-"], "enable-experimental-static-assert">,
   HelpText<"Enable experimental #assert">;
 
+def enable_experimental_async :
+  Flag<["-"], "enable-experimental-async">,
+  HelpText<"Enable experimental async concurrency">;
```

Comment:
Can you name this `-enable-experimental-concurrency`, since the feature is significantly broader?